.github/workflows/units-test-scripts-user-retirement.yml

```diff
@@ -0,0 +1,33 @@
+name: units-test-scripts-user-retirement
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [ '3.8' ]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
```

Comment:
We're getting warnings that these v2 actions are deprecated, can you update them to v4?